bots/zephyr_mirror_backend.py

```diff
@@ -1004,6 +1041,7 @@ def parse_args():
     return parser.parse_args()
 
 def die_gracefully(signal, frame):
+    # type: (Union[list, None, object], FrameType) -> None
```

Comment:
I thought signal should be something like Union[int, Tuple[int, bool]]
You may need to go digging into the python source to figure this out. None and list sound like bugs to me.